frontend/src/pages/Users/UserEmptyStates.js

```diff
@@ -0,0 +1,59 @@
+import Paper from "@material-ui/core/Paper";
```

Comment:
rename to `UsersGroupsEmptyStates.js` to make it clear that empty states for users and groups are defined here.